### ISSUE TITLE
An existing connection was forcibly closed by the remote host

### ISSUE THREAD

**Issue by SiIason** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#66]
Could be a blip as never seen this error before, just thought I'd post for into/tracking
```
09/02 12:36:23 - Error while trying to watch a minute: HTTPSConnectionPool(host='video-edge-94aedc.pdx01.abs.hls.ttvnw.net', port=443): Max retries exceeded with url: /v1/segment/CnXzOT7d0EYLrVqAZcYHIFRPZQbqUOY4MSkJ41rRErJElFPYBfVWa1UOuZOztYwrGtarMCoWpGC1sRPYYoTeUw6kN7VYLVcjAVoG6GW30FXoBnaqpWyhEmVasE_ACHB--gKzveuFd5paw0RBgE-9FQTWiLE9kO8B-nRt2C2Iwf5kwvJpbIhPQ3hRvy--021nAemyOtCWoTWss_k7wCStLiQt4-JDK6ltAp_9pYprpsoQ0xdqwtT0nO1a9jF3ym4qu_P80ab46VZ1l4owsu2ffV8GcDinTB4S1MKbvGfpcc_s5MdApmUFVlil7vqWPUUMiy_EsBVBxN7BZ5scG5Pl0oKccS2kmwT_FrG8zN92kta6uKRUx7uZtD3PtFyPDiFaxdVaf_ewkgeHGsgX2RcasuTNKNwQ6x79ZFTJZA0qEvUX2bxQCx5FI5lWGoQ1qPtrldNrzmU0J34sBcjqRL2MmSiVffT-Foym5uoHyTpexQUvc8pggyhuRl1jxuDcSxq3PEUfmZy2jSgfUPhI0W4V8V0X0n5DmIkUT99r8H7eqEmJYw4k8CNsDLq4P1s9gVpCCGqY-DgHt1ax8521x6SMt0nhGCR8Vg339zDsU7_ejf2l1KMI8LGGz2Ik2U4CSK6eeNXXtlk3h5Xn9WvFqWi6bP5foGrgqOO-L0qUDqaDU5yU1KCdl9Zf2xkrLL6pE6JOxKQkQXUNa6wOhzWn3Z_c7Ny1y6PjXpF4WZhBSQX-0yipI5UBLPgo-AGJV.ts (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025D1967E190>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))
09/02 12:36:27 - #0 - WebSocket error: [WinError 10054] An existing connection was forcibly closed by the remote host
09/02 12:36:27 - #0 - WebSocket closed
09/02 12:36:27 - #0 - Reconnecting to Twitch PubSub server in 30 seconds
09/02 12:37:21 - +12 --> scazzaf (3.06k points) - Reason: WATCH.
09/02 12:37:21 - +60 --> scazzaf (3.12k points) - Reason: CLAIM.
09/02 12:38:51 - +10 --> nazori (2.58k points) - Reason: WATCH.
09/02 12:38:52 - +50 --> nazori (2.63k points) - Reason: CLAIM.
09/02 12:40:30 - Error while trying to watch a minute: HTTPSConnectionPool(host='video-edge-94aedc.pdx01.abs.hls.ttvnw.net', port=443): Max retries exceeded with url: /v1/segment/CnXzOT7d0EYLrVqAZcYHIFRPZQbqUOY4MSkJ41rRErJElFPYBfVWa1UOuZOztYwrGtarMCoWpGC1sRPYYoTeUw6kN7VYLVcjAVoG6GW30FXoBnaqpWyhEmVasE_ACHB--gKzveuFd5paw0RBgE-9FQTWiLE9kO8B-nRt2C2Iwf5kwvJpbIhPQ3hRvy--021nAemyOtCWoTWss_k7wCStLiQt4-JDK6ltAp_9pYprpsoQ0xdqwtT0nO1a9jF3ym4qu_P80ab46VZ1l4owsu2ffV8GcDinTB4S1MKbvGfpcc_s5MdApmUFVlil7vqWPUUMiy_EsBVBxN7BZ5scG5Pl0oKccS2kmwT_FrG8zN92kta6uKRUx7uZtD3PtFyPDiFaxdVaf_ewkgeHGsgX2RcasuTNKNwQ6x79ZFTJZA0qEvUX2bxQCx5FI5lWGoQ1qPtrldNrzmU0J34sBcjqRL2MmSiVffT-Foym5uoHyTpexQUvc8pggyhuRl1jxuDcSxq3PEUfmZy2jSgfUPhI0W4V8V0X0n5DmIkUT99r8H7eqEmJYw4k8CNsDLq4P1s9gVpCCGqY-DgHt1ax8521x6SMt0nhGCR8Vg339zDsU7_ejf2l1KMI8LGGz2Ik2U4CSK6eeNXXtlk3h5Xn9WvFqWi6bP5foGrgqOO-L0qUDqaDU5yU1KCdl9Zf2xkrLL6pE6JOxKQkQXUNa6wOhzWn3Z_c7Ny1y6PjXpF4WZhBSQX-0yipI5UBLPgo-AGJV.ts (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025D1967EBB0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))
```

Script continued fine after this.

**Comment by Tkd-Alex** [comment]
Maybe It's just a connection errors :)
Keep tracking

**Comment by Tkd-Alex** [comment]
This error It's on master branch? :thinking:

**Comment by SiIason** [comment]
Yes, master branch.

```
10/02 16:36:23 - #0 - WebSocket error: [WinError 10054] An existing connection was forcibly closed by the remote host
10/02 16:36:23 - #0 - WebSocket closed
10/02 16:36:23 - #0 - Reconnecting to Twitch PubSub server in 30 seconds
10/02 16:36:34 - #1 - WebSocket error: [WinError 10054] An existing connection was forcibly closed by the remote host
10/02 16:36:34 - #1 - WebSocket closed
10/02 16:36:34 - #1 - Reconnecting to Twitch PubSub server in 30 seconds
10/02 16:36:53 - error from callback <function WebSocketsPool.on_close at 0x00000256DEB07940>: socket is already closed.
  File "<removed>\lib\site-packages\websocket\_app.py", line 346, in _callback
    callback(self, *args)
  File "<removed>\Twitch-Channel-Points-Miner-v2\TwitchChannelPointsMiner\classes\WebSocketsPool.py", line 100, in on_close
    WebSocketsPool.handle_reconnection(ws)
  File "<removed>\Twitch-Channel-Points-Miner-v2\TwitchChannelPointsMiner\classes\WebSocketsPool.py", line 118, in handle_reconnection
    self.submit(topic)
  File "<removed>\Twitch-Channel-Points-Miner-v2\TwitchChannelPointsMiner\classes\WebSocketsPool.py", line 43, in submit
    self.ws[-1].listen(topic, self.twitch.twitch_login.get_auth_token())
  File "<removed>\Twitch-Channel-Points-Miner-v2\TwitchChannelPointsMiner\classes\TwitchWebSocket.py", line 48, in listen
    self.send({"type": "LISTEN", "nonce": nonce, "data": data})
  File "<removed>\Twitch-Channel-Points-Miner-v2\TwitchChannelPointsMiner\classes\TwitchWebSocket.py", line 57, in send
    super().send(request_str)
  File "<removed>\lib\site-packages\websocket\_app.py", line 154, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "<removed>\lib\site-packages\websocket\_core.py", line 253, in send
    return self.send_frame(frame)
  File "<removed>\lib\site-packages\websocket\_core.py", line 279, in send_frame
    l = self._send(data)
  File "<removed>\lib\site-packages\websocket\_core.py", line 449, in _send
    return send(self.sock, data)
  File "<removed>\lib\site-packages\websocket\_socket.py", line 135, in send
    raise WebSocketConnectionClosedException("socket is already closed.")
10/02 16:37:25 - #2 - WebSocket error: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
10/02 16:37:25 - #2 - WebSocket closed
10/02 16:37:25 - #2 - Reconnecting to Twitch PubSub server in 30 seconds
```

**Comment by Tkd-Alex** [comment]
This kind of errors should be fixed on #59
